privacyidea/lib/crypto.py

```diff
@@ -108,15 +109,17 @@ def hmac_digest(self, data_input, hash_algo):
         return h
 
     def aes_decrypt(self, data_input):
```

Comment:
We wondered why this is using ECB. Should we rename this function to ``aes_ecb_decrypt``, so that we know this is done intentionally?

Reply:
makes sense.